src/kvstore/raftex/RaftPart.cpp

```diff
@@ -219,8 +219,7 @@ RaftPart::RaftPart(ClusterID clusterId,
         , ioThreadPool_{pool}
         , bgWorkers_{workers}
         , executor_(executor)
-        , snapshot_(snapshotMan)
-        , weight_(1) {
```

Comment:
Can `weight_` be deleted?

Reply:
Yeah, `weight_` is used to control elect interval before, if log is not more up-to-date than others, it will double interval (There is a log, we can rarely see it). After this PR, if I am explicitly rejected, just step down as follower, will try start election later.